client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -306,23 +308,25 @@ export class SeoForm extends React.Component {
 
 		const generalTabUrl = getGeneralTabUrl( slug );
 
-		const upsellProps = siteIsJetpack
-			? {
-					title: translate( 'Boost your search engine ranking' ),
-					feature: FEATURE_SEO_PREVIEW_TOOLS,
-					href: `/checkout/${ slug }/${ PRODUCT_UPSELLS_BY_FEATURE[ FEATURE_ADVANCED_SEO ] }`,
-			  }
-			: {
-					title: translate(
-						'Boost your search engine ranking with the powerful SEO tools in the Business plan'
-					),
-					feature: FEATURE_ADVANCED_SEO,
-					plan:
-						selectedSite.plan &&
-						findFirstSimilarPlanKey( selectedSite.plan.product_slug, {
-							type: TYPE_BUSINESS,
-						} ),
-			  };
+		const upsellProps =
+			siteIsJetpack && ! isFreeWPCOM
```

Comment:
Non-blocking: In the future, Atomic sites could support Personal and Premium plans, which are not eligible for the SEO feature. Would it be more robust to check `! isAtomic` here to avoid potential issues?

Reply:
That's a good idea, changed here: https://github.com/Automattic/wp-calypso/pull/53196/commits/69530633390ae28bd73864fc4d486844661f2423